docs/advanced-features/s3-prefix-permissions.md

```diff
@@ -36,3 +41,37 @@ Create "custom" role or policy with these permissions:
     ]
 }
 ```
+<!-- markdownlint-restore -->
+
+## Using the `Condition` element
+
+Alternatively, you can define granular permissions using the
+`Condition` element and [IAM policy
+string operators](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_variables.html):
+
+<!-- markdownlint-disable -->
+```json
+            ...
+            "Resource": [
+                "arn:aws:s3:::<BUCKET>",
+            ],
+            "Condition": {
+                "StringLike": {
+                    "s3:prefix": ["", ".quilt/*", "<PREFIX>/*"]
+                },
+                "StringEquals": {
+                    "s3:prefix": ["", ".quilt/", "<PREFIX>/"],
+                    "s3:delimiter": ["/"]
+                }
+            },
+            ...
```

Comment:
duplicate from https://github.com/quiltdata/quilt/pull/3295#discussion_r1094045347

1. Have you tested this? AFAICT this won't work
2.  if we could come up with a policy more strict than one above and still have catalog working, why'd we want still document both?